site/en_NZ.all.json

```diff
@@ -1,4 +1,5 @@
 {
+  "site_owner": { "other": "S72 TVOD" },
```

Comment:
What is this for?

Reply:
The `acceptinvite_page_header` string just says "Welcome to " so `site_owner` completes the sentence.  I got this way of doing it from the ABC template... I can change it if you want?